tests/unit/common/test_expressions.py

```diff
@@ -101,3 +108,38 @@ def test_additional_context(self):
     def test_raise_on_non_boolean_result(self):
         with pytest.raises(InvalidEvaluationResult):
             evaluate(Expression(statement="1"))
+
+
+class TestManagedExpressions:
+    def test_filter_supported_question_types(self, factories):
```

Comment:
Another test asserting that the same question isn't returned by `get_supported_form_questions` would be good 🙇